components/navbar/CreateDropdown.vue

```diff
@@ -7,48 +7,48 @@
         </div>
       </template>
       <b-dropdown-item has-link>
-        <b-tooltip
-          position="is-left"
+        <NeoTooltip
```

Comment:
tooltip overflows

![image](https://github.com/kodadot/nft-gallery/assets/22791238/e9188b65-cee2-4ee2-8885-7e0f33e698ba)


![image](https://github.com/kodadot/nft-gallery/assets/22791238/6de6c68b-7c5f-48d7-af6c-1c0ac52f4f5f)

Reply:
I made a change to have it multiline but I find it change the UI, should I keep it multiline?

Reply:
Either that or change max width of tooltip to large value to avoid it overflowing